Documentation/cloud-config.md

```diff
@@ -41,20 +41,49 @@ CoreOS tries to conform to each platform's native method to provide user data. E
 
 #### etcd
 
-The `coreos.etcd.*` parameters will be translated to a partial systemd unit acting as an etcd configuration file.
+The `coreos.etcd.*` and `coreos.etcd2.*` parameters will be translated to a partial systemd unit acting as an etcd configuration file.
 If the platform environment supports the templating feature of coreos-cloudinit it is possible to automate etcd configuration with the `$private_ipv4` and `$public_ipv4` fields. For example, the following cloud-config document...
 
 ```yaml
 #cloud-config
 
 coreos:
+  etcd2:
+    # generate a new token for each unique cluster from https://discovery.etcd.io/new
+    discovery: https://discovery.etcd.io/<token>
+    # multi-region and multi-cloud deployments need to use $public_ipv4
+    advertise-client-urls: http://$public_ipv4:2379
+    initial-advertise-peer-urls: http://$private_ipv4:2380
+    # listen on both the official ports and the legacy ports
+    # legacy ports can be omitted if your application doesn't depend on them
+    listen-client-urls: http://0.0.0.0:2379,http://0.0.0.0:4001
+    listen-peer-urls: http://$private_ipv4:2380,http://$private_ipv4:7001
+```
+
+...will generate a systemd unit drop-in like this:
```

Comment:
Now with them side-by side perhaps `will will generate a systemd unit drop-in for etcd2.service like this` and similarly for the etcd.service dropin.